job-descriptions/product-manager.md

```diff
@@ -0,0 +1,74 @@
+
+To create a product that serves the needs of all developers, we are building a diverse remote-first team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+You'll work with developers and other team members (from sales, marketing, and design) to plan, ship, and grow specific areas of our product. As an early member of our product team, you'll have significant product ownership and will communicate directly with our most sophisticated and demanding users to help preserve our market leadership.
+
+Why this role is special:
+
+- Our product has [extremely high engagement](https://engineeringblog.yelp.com/2019/11/winning-the-hackathon-with-sourcegraph.html) relative to most products (especially enterprise products). We get enormous amounts of user feedback from our unusually thoughtful user base.
+- Our developers use the product, so they care deeply about product matters. You won't be fighting apathy.
+
+Our [public roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit) shows the product priorities for each engineering team and we use [OKRs](https://about.sourcegraph.com/company/okrs) to communicate company goals for each quarter.
+
+#### Responsibilities
+
+You will:
+
+- Own an area of the product, and create and drive its product roadmap.
+- Collaborate with engineers and designers to ideate, plan, and build the solutions to customer problems.
+- Gather customer feedback to validate priorities, improve documentation, and define product requirements.
+- Be transparent in your internal and external communication. Sourcegraph is open source, and most discussion occurs in public, or in channels where you communicate directly with our customers.
+- Be a passionate advocate for a delightful user experience.
+
+#### Qualifications
+
+- Experience working as a software engineer.
+- Strong understanding of Git and Git workflows.
+- Excellent written and verbal communication skills, technical writing ability, and use of data to back up your arguments.
+- Ability to stay focused on goals and strategically prioritize work.
+- Deep passion for developer tools and productivity enhancements.
+- Demonstrated leadership skills and ability to drive and influence product strategy across engineering, design, and distribution.
+
+#### Nice to haves
+
+- Previous product management experience on a software development or developer tools product.
+- Experience building and/or contributing to developer tools.
+- Experience working remotely with globally distributed teams.
+
+### Location
+
+We are a remote-first, globally distributed team focused on results and employee happiness, so wherever you choose to work is up to you.
```

Comment:
This diverges a bit from what we have in other job descriptions and it would be nice to standardize.

I like your wording but wonder if it would avoid the problem mentioned in https://github.com/sourcegraph/careers/pull/34

Reply:
I just updated to explicitly state location and timezone don't matter. Do you like the updated wording enough for me to update on all other job descriptions?